core/src/test/scala/co.blocke.scalajack/v4/ParseOrElseTests.scala

```diff
@@ -7,26 +7,26 @@ import scala.language.postfixOps
 import scala.util.Try
 import org.joda.time.{ DateTime, DateTimeZone }
 import org.joda.time.format.DateTimeFormat
+import scala.reflect.runtime.universe.{ Type, typeOf }
 
 case class DefaultBlah() extends Blah
 
 class ParseOrElseTests extends FunSpec {
 
-  val sjJS = ScalaJack()
-  val vc = VisitorContext().copy(parseOrElse = Map("co.blocke.scalajack.test.v4.Blah" -> "co.blocke.scalajack.test.v4.DefaultBlah"))
+  val sjJS = ScalaJack().parseOrElse((typeOf[Blah] -> typeOf[DefaultBlah]))
```

Comment:
I like being able to use `typeOf[...]` when configuring `ScalaJack`, rather than the fully-qualified class names. 👍 It's even more flexible because `typeOf[...]` can handle classes with their own type parameters (`typeOf[List[String]]`).